include/bluetooth/services/ots.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Nordic Semiconductor ASA
+ * Copyright (c) 2020 - 2022 Nordic Semiconductor ASA
```

Comment:
I think this is the typical way of doing it

```suggestion
 * Copyright (c) 2020-2022 Nordic Semiconductor ASA
```